xml/Microsoft.Vsa/BaseVsaSite.xml

```diff
@@ -204,9 +204,9 @@
         <Parameter Name="optional" Type="System.Object" />
       </Parameters>
       <Docs>
-        <param name="notify">To be added.</param>
-        <param name="optional">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="notify">Notifies the host about events generated by the script engine.</param>
```

Comment:
I'll see if I can get access to the source here. Ideally notify and optional would have their own descriptions.

Reply:
what should I do about this? 
@mairaw

Reply:
Let's keep as-is. I'm not finding the source for this.